app/config.php

```diff
@@ -14,11 +15,16 @@
 use PhpParser\Parser;
 use PhpParser\ParserFactory;
 use PhpSchool\PhpWorkshop\Check\CgiOutputCheck;
+use PhpSchool\PhpWorkshop\Check\CodeParseCheck;
+use PhpSchool\PhpWorkshop\CodeModification as M;
```

Comment:
Yeah, i'm gonna rename it to `CodeInsertion` anyway, it doesn't really modify code